sandbox-backend/src/main/java/com/exadel/sandbox/team5/mapper/MapperUtil.java

```diff
@@ -0,0 +1,22 @@
+package com.exadel.sandbox.team5.mapper;
+
+import org.modelmapper.ModelMapper;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.List;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+@Configuration
+public class MapperUtil {
+
+    @Bean
+    public ModelMapper getMapper() {
+        return new ModelMapper();
+    }
+
+    public static <R, E> List<R> convertList(List<E> list, Function<E, R> converter) {
```

Comment:
We can use some sort of generic class ModelMapper<E, DTO> instead of that class.